README.md

```diff
@@ -0,0 +1,53 @@
+A tree-sitter grammar for Pkl.
+
+    # install dependencies
+    $ npm install
+
+    # build parser
+    $ npm run build
+
+    # parse some code
+    $ ./node_modules/.bin/tree-sitter parse test.pkl
+
+# Tests
+
+Tree sitter comes with its own test framework. Files in `corpus/`
+describe one test each. All tests in `corpus/` are performed by the
+command
+
+    $ tree-sitter test
+
+The script `src/synctests.ts` creates `corpus/` test files from tests in
+`pkl`'s `LanguageSnippetTests`. The script assumes that `../pkl` is a
+checked out `pkl` repository (and that the tests are under
+`pkl-core/src/test/files/LanguageSnippetFiles/input`). A pre-condition
+of the script is that `tree-sitter test` currently passes. If broken
+tests need replacing, the broken tests must first be deleted.
+
+It can be executed by running
+
+    $ npm run synctests
+
+# Upgrading tree-sitter
+
+Upgrading tree-sitter involves upgrading the NPM package.
+
+1.  Run `npm update tree-sitter` to install the newer version of tree-sitter.
+2.  Commit to main, and push.
+
+# Releasing
```

Comment:
It's better to remove this section or just point to the [wiki](https://github.com/apple/tree-sitter-pkl/wiki/Release-process) for how-to release.